test/functional/page_objects/visual_builder_page.ts

```diff
@@ -101,14 +101,18 @@ export function VisualBuilderPageProvider({ getService, getPageObjects }: FtrPro
     public async getMarkdownTableVariables(): Promise<
       Array<{ key: string; value: string; selector: any }>
     > {
-      const testTableVariables = await testSubjects.find('tsvbMarkdownVariablesTable');
+      const testTableVariables = await retry.try(
```

Comment:
@dmlemeshko this and the next `retry.try()` shouldn't be necessary right? If the default find timeout isn't long enough (though it probably should be) you can always extend the find timeout with the second argument to `testSubjects.find()`